docs/components/modeler/desktop-modeler/element-templates/defining-templates/c7-defining-templates.md

```diff
@@ -1,10 +1,11 @@
 ---
-id: defining-templates
-title: Defining templates
+id: c7-defining-templates
+title: Defining templates in Camunda Platform 7
+sidebar_label: Camunda 7
```

Comment:
We try to spell out our platform name completely for brand recognition and clarity, but that would look awkwardly verbose in the sidebar UI.

<img width="290" alt="image" src="https://github.com/camunda/camunda-platform-docs/assets/1051363/9e80407c-1341-4d36-9387-c01b58ea60b1">

Instead of nesting these, you can remove the "Defining templates" folder/directory. In the sidebar, you would have the "Defining templates" page, implied C8, and the "Defining templates in Camunda 7" page, explicitly C7. This would also reverse the ordering (C8 first) and match our experience in other places, like Best Practices.

<img width="290" alt="image" src="https://github.com/camunda/camunda-platform-docs/assets/1051363/ed9b4342-f105-4d50-b31d-27910a638a62">

(In the spirit of matching experience, you can drop "Platform"  in the sidebar as we do in Best Practices. I realize the screenshot is a contradiction. ) 

If you think we will have more content in the "Defining templates" directory in the future, keep it.